fli-iam/shanoir-ng
# [Do not merge] Shanoir issue#2141 log unification elk

How to test:
- Make an import and check logs -> Users should appear

Overall, all logs in application should look the same.
